### PR TITLE
MongoDB

### PR DESCRIPTION
**Pull request by sugr4**
Edit mongo conf for tailing MongoDB log messages
Edit mongostat_lib to fix a bug

@Microsoft/omsagent-devs

### CODE REVIEW

File: installer/conf/omsagent.d/mongo.conf [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  format /(?<timestamp>[^ ]*) (?<severity>[A-Z]) (?<component>[^ ]*)  \[(?<context>[^\]]*)\] (?<message>.*)/
+</source>
+
+<match oms.api.**>
+  type out_oms_api
+  log_level info
+
+  buffer_chunk_limit 5m
+  buffer_type file
+  buffer_path /var/opt/microsoft/omsagent/state/out_oms_mongo*.buffer
```
Comment (robbiezhang) [inline installer/conf/omsagent.d/mongo.conf]: to avoid dup prefix, can we name the buffer file as out_oms_api_workload_mongo*.buffer?

**Comment by robbiezhang** [comment]
Signoff

File: installer/conf/omsagent.d/mongo.conf [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<source>
+  type mongostat
+  tag oms.mongo
+  host localhost
+  port 27017
+  run_interval 5
```
Comment (agup006) [inline installer/conf/omsagent.d/mongo.conf]: Let's change the default to 10seconds

**Comment by sugr4** [comment]
changed run_interval to 5 sec, added Log.error_once if mongostat is not installed

**Pull request closed without merging**